runtime/src/lib.rs

```diff
@@ -1036,6 +1036,27 @@ impl pallet_staking::Config for Runtime {
     type BenchmarkingConfig = StakingBenchmarkingConfig;
 }
 
+parameter_types! {
+    pub const Deposit: u128 = DOLLARS;
+    pub const BatchSize: u32 = 64;
+    pub const MaxErasToCheckPerBlock: u32 = 1;
+}
+
+// fast unstake
+impl pallet_fast_unstake::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type Currency = Balances;
+    type Deposit = Deposit;
+    type ControlOrigin = EitherOfDiverse<
```

Comment:
Please, check this. Maybe it's better to delegate it to technical committee than council.